components/Navigation/Navigation.js

```diff
@@ -1,22 +1,126 @@
 import Link from "next/link";
+import { useRouter } from "next/router";
 import styled from "styled-components";
+import Image from "next/image";
+import BookmarkIcon from "assets/bookmarkicons/BookmarkPinIcon.svg";
 
 export default function Navigation() {
+  const router = useRouter();
+  const isOnBookmarksPage = router.pathname === "/bookmarks";
+
   return (
     <StyledNavigationBar>
-      <Link href="/">Home</Link>
-      <Link href="/bookmarks">My Emotions</Link>
+      <LinkStyled href="/bookmarks">
+        <IconWrapper isActive={isOnBookmarksPage}>
+          <StyledBookmarkIcon
+            src="/BookmarkPinIcon.svg"
+            alt="Bookmark Icon"
+            width={24}
+            height={24}
+          />
+          My Emotions
+        </IconWrapper>
+      </LinkStyled>
+      <CenterWrapper>
+        <Link href="/" passHref>
+          <LogoCanvas isOnBookmarksPage={isOnBookmarksPage}>
+            <Image src="/moodwave-logo.svg" alt="Logo" width={45} height={45} />
+            <HomeText>Home</HomeText>
+          </LogoCanvas>
+        </Link>
+      </CenterWrapper>
     </StyledNavigationBar>
   );
 }
 
 const StyledNavigationBar = styled.nav`
   display: flex;
-  justify-content: space-around;
+  justify-content: space-between;
+  align-items: center;
   padding: 10px;
-  background-color: #f9f9f9;
-  border-top: 1px solid #d3d3d3;
+  background-color: #e0e1f0;
   position: fixed;
   bottom: 0;
   width: 100%;
+  height: 3.75rem;
+  gap: 1.25rem;
+  border-top: 1px solid #e0e1f0;
+`;
+
+const CenterWrapper = styled.div`
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+  display: flex;
+  align-items: center;
+`;
+
+const LogoCanvas = styled.div`
+  color: #313366;
+  width: 3.5rem;
+  height: 3.5rem;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  background-color: ${(props) =>
+    props.isOnBookmarksPage ? "#E0E1F0" : "#fff"};
+  border-radius: 50%;
+  border: 1px solid
+    ${(props) => (props.isOnBookmarksPage ? "#E0E1F0" : "#313366")};
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  padding: 3.125rem;
+  transition: transform 0.3s ease;
+  transform: scale(1);
+
+  &:hover {
+    transform: scale(1.1);
+  }
+
+  @media (hover: none) and (pointer: coarse) {
+    &:hover {
+      transform: scale(1);
+    }
+  }
```

Comment:
I think you don't need this anymore

Reply:
Actually, we do need it. It defines some settings for the canvas on which the logo is placed. Once I remove it, it no longer appears correctly.